libs/gi/page-team/src/CharacterDisplay/Content.tsx

```diff
@@ -146,44 +147,58 @@ function TabNav({
       }}
     >
       {!hideTitle && (
-        <CardContent
-          sx={{
-            display: 'flex',
-            justifyContent: 'center',
-            p: 0,
-            textShadow: '#000 0 0 10px !important',
-            position: 'relative',
-          }}
-        >
-          <CardActionArea onClick={() => setEditMode(true)} sx={{ p: 2 }}>
+        <>
+          <CardActionArea onClick={() => setEditMode(true)}>
             <BootstrapTooltip
               title={
-                teamChar.description ? (
-                  <Typography>{teamChar.description}</Typography>
-                ) : undefined
+                <Box>
+                  <Box sx={{ display: 'flex', color: 'info.light', gap: 1 }}>
+                    <BorderColorIcon />
+                    <Typography>
+                      <strong>{t`loadout.editNameDesc`}</strong>
+                    </Typography>
+                  </Box>
+                  {!!teamChar.description && (
+                    <Typography>{teamChar.description}</Typography>
+                  )}
+                </Box>
               }
             >
-              <Typography
-                variant="h6"
+              <CardContent
                 sx={{
                   display: 'flex',
-                  gap: 1,
-                  alignItems: 'center',
                   justifyContent: 'center',
+                  position: 'relative',
+                  '&:hover': {
+                    color: 'info.light',
```

Comment:
Also, are we planning to add this hover to everything? I think the CardActionArea hover is enough, and matches the rest of the sites styling. We don't do color text hovering anywhere else afaik

Reply:
This is sort of one-off, since the background of the action area is sort of unique.